packages/shared/src/components/auth/OnboardingRegistrationForm.tsx

```diff
@@ -247,4 +291,113 @@ const OnboardingRegistrationForm = ({
   );
 };
 
+export const TermAndPrivacy = (): ReactElement => (
+  <Typography color={TypographyColor.Secondary} type={TypographyType.Footnote}>
+    By continuing, you agree to the{' '}
+    <Typography
+      className="underline hover:no-underline"
+      color={TypographyColor.Tertiary}
+      href={termsOfService}
+      rel="noopener"
+      tag={TypographyTag.Link}
+      target="_blank"
+    >
+      Terms of Service
+    </Typography>{' '}
+    and{' '}
+    <Typography
+      className="underline hover:no-underline"
+      color={TypographyColor.Tertiary}
+      href={cookiePolicy}
+      rel="noopener"
+      tag={TypographyTag.Link}
+      target="_blank"
+    >
+      Privacy Policy
+    </Typography>
+    .
+  </Typography>
+);
+
+const AlreadySignedLogin = ({ onClickLogin }: { onClickLogin: () => void }) => (
+  <Typography
+    className="mt-6"
+    color={TypographyColor.Secondary}
+    type={TypographyType.Callout}
+  >
+    Already have an account?{' '}
+    <ClickableText
+      className="!inline-flex !text-inherit"
+      defaultTypo={false}
+      inverseUnderline
+      onClick={onClickLogin}
+      type="button"
+    >
+      Log in
+    </ClickableText>
+  </Typography>
+);
+
+export const OnboardingRegistrationFormExperiment = ({
+  isReady,
+  onContinueWithEmail,
+  onExistingEmail,
+  onProviderClick,
+  targetId,
+  trigger,
+}: OnboardingRegistrationFormProps): ReactElement => {
+  const { logEvent } = useLogContext();
+  useEffect(() => {
+    logEvent({
+      event_name: AuthEventNames.OpenSignup,
+      extra: JSON.stringify({ trigger }),
+      target_id: targetId,
+    });
+    // @NOTE see https://dailydotdev.atlassian.net/l/cp/dK9h1zoM
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
```

Comment:
Is this a new event?

Reply:
Not new event, just the same event from control component ✔️

Reply:
Maybe just failsafe it with a ref to see if was already fired since it's in useEffect now.